fwl-expect/package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "fwl-expect",
+  "version": "1.0.0",
+  "description": "This is a jest-expect-message moded version for Sparta Courses",
+  "main": "index.js",
+  "repository": "git@github.com:fewlines-education/fwl-expect.git",
+  "author": "Fewlines",
+  "license": "MIT",
+  "private": false,
+  "dependencies": {
+    "@types/jest": "^25.1.4",
+    "@typescript-eslint/eslint-plugin": "^2.24.0",
+    "@typescript-eslint/parser": "^2.24.0",
+    "eslint": "^6.8.0",
+    "eslint-config-prettier": "^6.11.0",
+    "eslint-plugin-prettier": "^3.1.3",
+    "jest": "^25.1.0",
+    "jest-extended": "^0.8.1",
```

Comment:
what is that?

Reply:
Leftovers when trying to find Matchers type for ts! 
Will get rid of it :)